pages/api_integration-guides/how_to_transfer.mdx

```diff
@@ -0,0 +1,214 @@
+
+Example: withdrawing 100 USDC from subaccount number `0`.
+
+<Tabs items={["TypeScript", "Python"]}>
+<Tab>
+```typescript copy
+import { FaucetClient } from "@dydxprotocol/v4-client-js";
+
+const NETWORK = <insert_network_here>;
+const USDC_AMOUNT = 100;
+const USDC_ASSET_ID = 0;
+const SUBACCOUNT_NUMBER = 0;
+
+const wallet = await LocalWallet.fromMnemonic(DYDX_MNEMONIC, BECH32_PREFIX);
+const client = await ValidatorClient.connect(NETWORK);
+
+const subaccount = new SubaccountInfo(wallet, SUBACCOUNT_NUMBER);
+
+const quantums = new Long(USDC_AMOUNT * 1_000_000);
+const tx = await client.post.withdraw(subaccount, USDC_ASSET_ID, quantums);
+```
+</Tab>
+<Tab>
+```python copy
+from v4_client_py import Subaccount
+
+NETWORK = <insert_network_here>
+USDC_AMOUNT = 100
+USDC_ASSET_ID = 0
+SUBACCOUNT_NUMBER = 0
+
+wallet = LocalWallet.from_mnemonic(DYDX_MNEMONIC, BECH32_PREFIX)
+client = ValidatorClient.connect(NETWORK)
+
+subaccount = Subaccount(wallet, SUBACCOUNT_NUMBER)
+
+quantums = USDC_AMOUNT * 1_000_000
+tx = client.post.withdraw(
+    subaccount,
+    wallet.address,
+    SUBACCOUNT_NUMBER,
+    USDC_ASSET_ID,
+    quantums,
+)
+```
+</Tab>
+</Tabs>
+
+## Transfer from subaccount to subaccount
+The `transfer` transaction must be used to perform this transfer.
+
+Parameters ([link](https://github.com/dydxprotocol/v4-chain/blob/main/proto/dydxprotocol/sending/transfer.proto#L13))
+
+Example: transferring 100 USDC from subaccount number `0` to `100`.
+
+<Tabs items={["TypeScript", "Python"]}>
+<Tab>
+```typescript copy
+import { FaucetClient } from "@dydxprotocol/v4-client-js";
+
+const NETWORK = <insert_network_here>;
+const USDC_AMOUNT = 100;
+const USDC_ASSET_ID = 0;
+const SUBACCOUNT_NUMBER_FROM = 0;
+const SUBACCOUNT_NUMBER_TO = 1;
+
+const wallet = await LocalWallet.fromMnemonic(DYDX_MNEMONIC, BECH32_PREFIX);
+const client = await ValidatorClient.connect(NETWORK);
+
+const subaccount = new SubaccountInfo(wallet, SUBACCOUNT_NUMBER_FROM);
+
+const quantums = new Long(USDC_AMOUNT * 1_000_000);
+const tx = await client.post.transfer(
+  subaccount,
+  wallet.address,
+  SUBACCOUNT_NUMBER_TO,
+  USDC_ASSET_ID,
+  quantums
+);
+```
+</Tab>
+<Tab>
+```python copy
+from v4_client_py import Subaccount
+
+NETWORK = <insert_network_here>
+USDC_AMOUNT = 100
+USDC_ASSET_ID = 0
+SUBACCOUNT_NUMBER_FROM = 0
+SUBACCOUNT_NUMBER_TO = 1
+
+wallet = LocalWallet.from_mnemonic(DYDX_MNEMONIC, BECH32_PREFIX)
+client = ValidatorClient.connect(NETWORK)
+
+subaccount = Subaccount(wallet, SUBACCOUNT_NUMBER_FROM)
+
+quantums = USDC_AMOUNT * 1_000_000
+tx = client.post.transfer(
+    subaccount,
+    wallet.address,
+    SUBACCOUNT_NUMBER_TO,
+    USDC_ASSET_ID,
+    quantums,
+)
+```
+</Tab>
+</Tabs>
+
+## Determining parameters
+
+* Asset
+Asset ID can be fetched using the `/dydxprotocol/assets/asset` endpoint. [Example](https://dydx-api.lavenderfive.com:443/dydxprotocol/assets/asset)
```

Comment:
nit: the way the Example link is formatted makes this a little bit unclear -- it could read "Example Collateral token (USCD)..."

Suggest putting the Example links in parentheses 

I took a quick look at docs pages on the same level and it doesn't seem like this formatting is standard, so I think it's worth changing

Reply:
done!